config/default.yml

```
@@ -4416,6 +4416,16 @@ Style/ItAssignment:
   Enabled: pending
   VersionAdded: '1.70'
 
+Style/ItBlockParameter:
+  Description: 'Check for replacement with or disallowing of the `it` block parameter.'
```

This doesn't sound quite right to me, maybe something like this (it's correct for 2 of the 3 styles)?

```suggestion
  Description: 'Checks for blocks with one argument where `it` can be used.'
```